spec/request/user_request_spec.rb

```diff
@@ -6,6 +6,28 @@
 RSpec.describe 'Fetching user data', type: :request do
   include SchemaMatchers
 
+  context 'GET /v0/user in camel - when an LOA 3 user is logged in' do
```

Comment:
copied from the context below, but changed to `in camel`

Reply:
adding the `headers: { 'X-Key-Inflection' => 'camel' }` was the only difference between the `before`-blocks